packages/contracts-core/contracts/manager/LightManager.sol

```diff
@@ -53,6 +68,30 @@ contract LightManager is AgentManager, InterfaceLightManager {
         __Ownable_init();
     }
 
+    // ════════════════════════════════════════════════ OWNER ONLY ═════════════════════════════════════════════════════
+
+    /// @inheritdoc InterfaceLightManager
+    function proposeAgentRootWhenStuck(bytes32 agentRoot_) external onlyOwner onlyWhenStuck {
+        // Update the proposed agent root, clear the timer if the root is empty
+        _proposedAgentRoot = agentRoot_;
+        _agentRootProposedAt = agentRoot_ == 0 ? 0 : block.timestamp;
```

Comment:
what's the logic for allowing owner to propose `agentRoot_ = 0` here if would then revert below when attempt to call `resolveProposedAgentRoot`?

Reply:
That's a good point. It was done for the purpose of cancelling the previously proposed one, but it's much cleaner to have a separate function for that. Plus might need to be able to cancel the proposed root if the chain became unstuck naturally.

Will work on a quick fix for that.